modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/ImageMosaicConfigHandler.java

```diff
@@ -1439,6 +1450,10 @@ public void updateConfiguration(GridCoverage2DReader coverageReader,
                 currentConfigurationBean.getCatalogConfigurationBean().setHeterogeneousCRS(true);
             }
 
+            // nodata treatment
+            
+
+
```

Comment:
Don't hate me for this review: why 3 whitelines? :)

Reply:
Because I like having space! (will fix ;-) )